app/app.go

```diff
@@ -299,25 +373,45 @@ func (app *GaiaApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci
 	var genesisState simapp.GenesisState
 	app.cdc.MustUnmarshalJSON(req.AppStateBytes, &genesisState)
 
-	return app.mm.InitGenesis(ctx, app.cdc, genesisState)
+	res := app.mm.InitGenesis(ctx, app.cdc, genesisState)
+
+	// Set Historical infos in InitChain to ignore genesis params
+	stakingParams := app.stakingKeeper.GetParams(ctx)
+	stakingParams.HistoricalEntries = 1000
+	app.stakingKeeper.SetParams(ctx, stakingParams)
```

Comment:
Note, we should change this default on the SDK before merging this PR

Reply:
https://github.com/cosmos/cosmos-sdk/pull/6059

Reply:
Merged.